vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
```

Comment:
**praise**: Great job including a link to the Vite configuration documentation. This is helpful for future developers who may need to understand or modify the configuration.